code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -127,6 +127,7 @@ GLOBAL_LIST_INIT(diamond_recipes, list ( \
 	sheettype = "uranium"
 	materials = list(MAT_URANIUM=MINERAL_MATERIAL_AMOUNT)
 	novariants = TRUE
+	grind_results = list("uranium" = 2)
```

Comment:
are these actually 2u of chems per sheet or am i missing something?

Reply:
There are, same for all the mineral sheet types.

Reply:
Shouldn't it be 20 per sheet to match current gameplay? I'm sure if you grind a chair's worth of metal you can get more than a few drops of iron

Reply:
Hm, I thought it was just a flat 20. If it's 20 currently, I can do that, sure.

Reply:
Right now, grinding produces 20u of chems per sheet. For instance, grinding one sheet of plasma in xenobiology produces 20 u of plasma, which is enough for 4 reactions. If this really results in 2 units of plasma, then xenobiology, chemistry, and virology are going to have issues.

Reply:
You need 1 unit of plasma for reaction to start, no? 20u of plasma is 20 reactions, if you are careful

Reply:
yeah but your going to want more that 2 reactions befor you need to bow and scrape to the guy running R&D/Robo